### ISSUE TITLE
nothing echoed on multiple import

### ISSUE THREAD

**Issue by paulp** [lihaoyi/Ammonite#39]
This is with deafdc774a, commit of 45 minutes ago.
```
@ import psp._, std._, api._, StdShow._, StdEq._

@
```
As opposed to
```
@ import psp._
import psp._
@
```
Good news is this works in initial test:
```
@ implicit def someShow[A](implicit z: Show[A]) = ammonite.pprint.PPrinter[A]((t, c) => Iterator(z show c))
defined function someShow
@ Direct(1, 2, 3)
res6: psp.api.Direct[Int] = [ 1, 2, 3 ]
@
```

**Comment by lihaoyi** [comment]
I never even knew that was valid syntax. Learn something new every day...

**Comment by lihaoyi** [comment]
Should be fixed by 7521782dd3728437534b3b22c5dbfeecdec07af8
